platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -743,3 +744,8 @@ else
     fi
     "$APP_NAME"_check
 fi
+
+if ! echo "" | grep -E '^[0-9]+.?[0-9]+$'; then
```

Comment:
Do you plan to leave this test in the script?

Reply:
Removed. Thanks for catching this!